### PR TITLE
Create Basic CLI with Flags

### LINKED ISSUE #4: Need first implementation of CLI
First implementation of the CLI with a start command and basic options (port, etc.)

### PR DESCRIPTION
**Pull request by andrewnguyen22**
Fixes #4

### CODE REVIEW

**Comment by andrewnguyen22** [comment]
Will resolve conflict upon approval

File: cmd/util/const.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// This package is for command line specific utilities.
+package util
```
Comment (luyzdeleon) [inline cmd/util/const.go]: Should we have a project global constant modules? I feel like version and client identifier are constants that could be used elsewhere.

File: cmd/util/flags.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+func ParseFlags() {
+	flag.Parse()
+}
+
+/*
+"PrintClientInfo" prints startup information about the client
+ */
+func PrintClientInfo(){
+	fmt.Println("CLIENT ID:", clientIdentifier)
```
Comment (luyzdeleon) [inline cmd/util/flags.go]: This is fine for now, but I think there's a better way to retrieve these values. Maybe the CMD should build some sort of configuration object that gets created based on the CLI flags the user typed in, then we can just do `Println(Configuration.description())` or something along those lines.

**Review by luyzdeleon — COMMENTED** [review]
Minor comments.

File: cmd/util/const.go (hunk 1 above)
Comment (andrewnguyen22) [reply cmd/util/const.go]: Agreed will add

File: cmd/util/flags.go (hunk 2 above)
Comment (andrewnguyen22) [reply cmd/util/flags.go]: That's a good idea

Comment (andrewnguyen22) [reply cmd/util/flags.go]: See issue #9 and approve this PR

File: cmd/util/const.go (hunk 1 above)
Comment (andrewnguyen22) [reply cmd/util/const.go]: See [commit](https://github.com/pokt-network/pocket-core/commit/7c91b982f490fdb42aa4ab1f583dcb084621432a) for change

**Review by luyzdeleon — DISMISSED** [review]

**Review by luyzdeleon — DISMISSED** [review]

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**